Wasm/Bytes.lean

```diff
@@ -614,11 +652,12 @@ def mtob (m : Module) : ByteArray :=
   let types := extractTypes m
   let typeSection := if m.func.isEmpty then b0 else
     typeHeader ++ lindex (vectorise types)
+  let (funcBodies, blockLabels) := extractFuncBodies m
   magic ++
   version ++
   typeSection ++
   (extractFuncIds m types) ++
   (extractGlobals m.globals) ++
   (extractExports m) ++
-  (extractFuncBodies m) ++
-  (extractIdentifiers m)
+  (funcBodies) ++
```

Comment:
Imagine we had `Control.Foldl` and just describe this computation here and run it "fused" later on.